instrumentation/spring/starters/spring-boot-starter/build.gradle.kts

```diff
@@ -12,7 +12,10 @@ dependencies {
   api("org.springframework.boot:spring-boot-starter:$springBootVersion")
   api("org.springframework.boot:spring-boot-starter-aop:$springBootVersion")
   api(project(":instrumentation:spring:spring-boot-autoconfigure"))
+  api(project(":instrumentation-annotations"))
   implementation(project(":instrumentation:resources:library"))
+  implementation("io.opentelemetry:opentelemetry-extension-trace-propagators")
+  implementation("io.opentelemetry.contrib:opentelemetry-aws-xray-propagator")
```

Comment:
It is an extension located in the Java contrib repo (https://github.com/open-telemetry/opentelemetry-java-contrib). So, it would not make sense to enable it from  a project located in the Java instrumentation Github repo.

Reply:
javaagent is also including it - and the docs mention it as one of the values.

Reply:
Could you please elaborate with links

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/d36d7df1843e4c618c72002f8a3e25101738d96a/javaagent-tooling/build.gradle.kts#L38

Reply:
Agent includes it because we have instrumentations that use this propagator. On the one hand including it is fine because the propagator is small and doesn't have dependencies. On the other hand getting rid of it later on might be difficult once the starter is stable. I'd personally not include it and I would also not include `opentelemetry-extension-trace-propagators`. The code in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/propagators/CompositeTextMapPropagatorFactory.java already treats these as optional dependencies, I see no compelling reason for changing this.
You may wish to raise this on the SIG meeting if you believe that these should be included.

Reply:
No, it's not a priority for me - we'll just document that.